### PR TITLE
UI: Rename “User Preferences” + Use ellipses

### PR DESCRIPTION
**Pull request by JohJakob**
- Rename “User Preferences” to “Preferences”
- Replace occurrences of `...` (three periods) with `…` (ellipsis) in user interface labels

### CODE REVIEW

File: src/gimelstudio/application.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def __init__(self):
         self.saveprojectfile_menuitem = flatmenu.FlatMenuItem(
             file_menu,
             id=wx.ID_ANY,
-            label="{0}{1}".format(_("Save Project..."), "\tCtrl+S"),
+            label="{0}{1}".format(_("Save Project…"), "\tCtrl+S"),
```
Comment (Correct-Syntax) [inline src/gimelstudio/application.py:97]: Just so I know, how are you creating these ellipses, @JohJakob ? Is it a keyboard shortcut, from the system special characters or something else? 🔠

Comment (JohJakob) [reply src/gimelstudio/application.py:97]: It’s a very convenient keyboard shortcut on Mac (`option` + `.`). On Windows, you can press `Alt` and then type the code 0133 to get an ellipsis.

Comment (Correct-Syntax) [reply src/gimelstudio/application.py:97]: Okay, thanks. I have just always used ... It works too. 🤣

Comment (JohJakob) [reply src/gimelstudio/application.py:97]: Yeah, I guess it only makes a difference when using a screen reader 😀

**Pull request merged**